prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -192,13 +215,16 @@ def test_connection(
 
         Args:
             kubeconfig_file (str): Path to the kubeconfig file.
+            kubeconfig_content (dict): Content of the kubeconfig file.
             input_context (str): Context name.
-
+            raise_on_exception (bool): Whether to raise an exception on error.
         Returns:
             Connection: A Connection object.
         """
         try:
-            KubernetesProvider.setup_session(kubeconfig_file, input_context)
+            KubernetesProvider.setup_session(
+                kubeconfig_file, kubeconfig_content, input_context
+            )
             client.CoreV1Api().list_namespace(timeout_seconds=2, _request_timeout=2)
```

Comment:
Would we need to add an optional `namespace` here for validation purposes?

Reply:
Listing namespaces is a simple and non-invasive Kubernetes API call. It confirms that the Kubernetes client (and thus, the user or service account) can establish a connection to the cluster’s API server, which is the primary indicator of a valid session.

Reply:
That's great but my point was focused on adding an specific `namespace` as an optional validation for the client, since we allow that argument to be used.